### DISCUSSION TITLE
does not work on hp reverb

### DISCUSSION THREAD

**Discussion by polybasss** [fholger/crysis_vrmod#11] [General]
I have tried version 2.0 and it works for me in quest2 but not in Hp reverb

**Comment by fholger** [comment]
In what way does it not work? Can you post the contents of the `Game.log`, please? Also, does it work if you set SteamVR as the active OpenXR runtime and go through that?

**Reply by polybasss** [reply]
Ok, it was the execution time that I hadn't changed, it's a pain when I switch between quest and reverb it changes and I don't always remember, now let's see if I can recenter the view, I already see that the steamvr button doesn't do anything , thank you so much
I edit: solved the recentering, as you describe from the desktop menu it works, what things....
